analyses/cms-open-data-ttbar/reana.yaml

```diff
@@ -0,0 +1,38 @@
+inputs:
+  files:
+    - ttbar_analysis_pipeline.ipynb
+    - ttbar_analysis_pipeline.py
+    - nanoaod_inputs.json
+    - nanoad_branch_ratios.json
+    - cabinetry_config.yml
+    - cabinetry_config_ml.yml
+    - coffea.ipynb
+    - corrections.json
+    - GetIOBranches.py
+    - jetassignment_training.ipynb
+    - jetassignment_training.py
+    - make_corrections_json.py
+  directories:
+    - histograms
+    - utils
+    - models
+    - reference
+workflow:
+  type: serial
+  resources:
+    kerberos: true
```

Comment:
`kerberos: True` could be bothersome for people that would be running this example outside of CERN.  So I guess we should leave it out, and just document (e.g. in the full run docs) that setting Kerberos helps with faster data access.